From: Christian Marillat <marillat@deb-multimedia.org>
Date: Thu, 6 Jul 2017 15:59:17 +0200
Subject: Fontconfig from system

Use system wide font config dir /etc/fonts
instead of the DroidSans packaged in-tree
---
 Engine/AppManager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Engine/AppManager.cpp b/Engine/AppManager.cpp
index 0a5cdf5..e9f2317 100644
--- a/Engine/AppManager.cpp
+++ b/Engine/AppManager.cpp
@@ -476,7 +476,7 @@ AppManager::loadFromArgs(const CLArgs& cl)
     // set fontconfig path on all platforms
     if ( qgetenv("FONTCONFIG_PATH").isNull() ) {
         // set FONTCONFIG_PATH to Natron/Resources/etc/fonts (required by plugins using fontconfig)
-        QString path = QCoreApplication::applicationDirPath() + QString::fromUtf8("/../Resources/etc/fonts");
+        QString path = QString::fromUtf8("/etc/fonts");
         QFileInfo fileInfo(path);
         if ( !fileInfo.exists() ) {
             std::cerr <<  "Fontconfig configuration file " << path.toStdString() << " does not exist, not setting FONTCONFIG_PATH "<< std::endl;
